Tabloid/Repositories/TagRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using Tabloid.Models;
+
+namespace Tabloid.Repositories
```

Comment:
Are there DbUtils methods that could be used in this repo to do some of this work? We'll want to use those utils methods consistently across the codebase.